db/migrate/20220103143414_add_videoids_to_artists.rb

```diff
@@ -0,0 +1,7 @@
+class AddVideoidsToArtists < ActiveRecord::Migration[6.1]
```

Comment:
nitpick: `ids`, `I` should be capitalized

Reply:
Fixed!